pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+python_files = test_*.py
+python_functions = test_*
```

Comment:
yes in `pytest.ini` these are required because these two lines are selecting all the files with test names and then applying the ignore command on the tests.